<launch>
<arg name="path_to_config" />
<node name="wav_player" pkg="esiaf_wav_player" type="start_wav_player.py" output="screen" args="$(arg path_to_config)" />
</launch>